synapse/axon.py

```diff
@@ -1358,10 +1358,9 @@ async def readlines(self, sha256):
         feedtask = None
 
         try:
-            todo = s_common.todo(_spawn_readlines, sock00)
             async with await s_base.Base.anit() as scope:
 
-                scope.schedCoro(s_coro.spawn(todo, log_conf=await self._getSpawnLogConf()))
+                scope.schedCoro(s_coro.forked(_spawn_readlines, sock00))
```

Comment:
The use of this in a generator needs to be discussed. It's possible for this to lock up an Axon's ability to service readlines/csvrows apis. For example, running with SYN_FORKED_WORKERS=2, attempting to call this API > 2 times causes the subsequent calls to lock up. For example:

```
>>> genr0 = prox.readlines(fhash)
>>> genr1 = prox.readlines(fhash)
>>> genr2 = prox.readlines(fhash)
>>> genr3 = prox.readlines(fhash)
>>> for l0 in genr0:
...  print(f'{l0=}')
...  for l1 in genr1:
...   print(f'{l1=}')
...   for l2 in genr2:
...    print(f'{l2=}')
...    for l3 in genr3:
...     print(f'{l3=}')
... 
l0='hehehahahehehahahehehahahehehahahehehaha'
l1='hehehahahehehahahehehahahehehahahehehaha'

... # No more lines are presented
```

Reply:
Reverted axon to use spawn